dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -1956,6 +1956,79 @@ global using nfloat = global::System.Runtime.InteropServices.NFloat%3B
 		</ItemGroup>
 	</Target>
 
+	<Target Name="_CopyLocalBindingResources" AfterTargets="ResolveAssemblyReferences" BeforeTargets="CopyFilesToOutputDirectory">
+		<!--
+
+			We need to copy binding resource packages (either zipped or as a
+			.resources bundle) together with the binding assembly when the
+			binding assembly is copied around. This typically happens when
+			another library project references the binding project, in which
+			case the ResolveAssemblyReference task will add the binding
+			library to the ReferenceCopyLocalPaths item group (when building
+			the library project), which will cause the binding assembly to be
+			copied to the output directory for the other library project.
+
+			If this isn't done, then any projects referencing the library
+			project (and not the binding project), won't see any binding
+			resource packages from the binding project.
+
+			This is not good (https://github.com/xamarin/xamarin-macios/issues/13910).
+
+			This is a target that runs after the ReferenceCopyLocalPaths item
+			group has been populated by the ResolveAssemblyReferences target,
+			and will add any binding resource packages to the
+			ReferenceCopyLocalPaths item group.
+
+			This is somewhat complicated by the fact that we can have
+			.resources bundles (i.e. directories), so we need to expand those
+			to all their contained files.
+
+		-->
+
+		<ItemGroup>
+			<!-- List all potential compressed binding resource packages next to the assemblies we're copying to the output directory -->
+			<_CompressedBindingPackagesFromReferencedAssembliesCandidates Include="@(ReferenceCopyLocalPaths -> '%(RootDir)%(Directory)%(Filename).resources.zip')" />
+			<!-- We only care about those that actually exist, and we don't want duplicates either -->
+			<_CompressedBindingPackagesFromReferencedAssemblies Include="@(_CompressedBindingPackagesFromReferencedAssembliesCandidates->Distinct())" Condition="Exists('%(Identity)')" />
+
+			<!-- List all potential binding resource packages next to the assemblies we're copying to the output directory -->
+			<_BindingPackagesFromReferencedAssembliesDirectoriesCandidates Include="@(ReferenceCopyLocalPaths -> '%(RootDir)%(Directory)%(Filename).resources')" />
+			<!-- We only care about those that actually exist, and we don't want duplicates either -->
+			<_BindingPackagesFromReferencedAssembliesDirectoriesExists Include="@(_BindingPackagesFromReferencedAssembliesDirectoriesCandidates->Distinct())" Condition="Exists('%(Identity)')" />
+		</ItemGroup>
+
+		<!--
+			We need to expand items in an item group using globs, which is
+			kind of tricky, because globs in item transformations aren't
+			treated as globs. The workaround is to use a custom task for this.
+
+			Note that this task should run on Windows for remote builds (so we're not conditioning this task on IsMacEnabled).
+		-->
+		<GetFileSystemEntries
+			SessionId="$(BuildSessionId)"
```

Comment:
If this should always run locally (in both Windows and macOS), we should not set the `SessionId` property. 

I won't make a big difference now because the task does not support running remotely, but since that property is what we use to decide where to execute the task, it could be a problem in the future if we ever decide to support running it remotely.

Reply:
My comment is somewhat wrong: we should execute this task remotely when connected to a remote Mac, but locally on Windows when not connected to a remote Mac (for hot restart for instance).

I'll change the comment to make it clearer.

Also the task supports remote execution as far as I can see: https://github.com/xamarin/xamarin-macios/blob/6300a6aff7e65a36abdad47056b5cfe95f88b9f5/msbuild/Xamarin.MacDev.Tasks/Tasks/GetFileSystemEntries.cs#L8